api/client/volume/create.go

```diff
@@ -26,10 +26,13 @@ func newCreateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "create",
+		Use:   "create [VOLUME_NAME]",
 		Short: "Create a volume",
-		Args:  cli.NoArgs,
+		Args:  cli.RequiresMaxArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 1 {
```

Comment:
Need some validation: the user should only be able to specify `--name` or provide the positional arg, but not both.